### PR TITLE
Adding starting and stopping database state

### PR DESCRIPTION
**Pull request by gfx54b**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/administration/databases.adoc [hunk 1]
```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
...
 
 | currentStatus
 | The actual status of the database. label:default-output[]
+
+The value can be one of the following:
+
+* `online` - The database is running.
+* `offline` - The database is not running.
+* `starting` - The database is not running, but is about to be.
+* `stopping` - The database is not running anymore, but still not stopped completely.
```
Comment (jackwaudby) [inline modules/ROOT/pages/administration/databases.adoc]: Same here, I think you need a has between the but and still

File: modules/ROOT/pages/administration/databases.adoc [hunk 2]
```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
...
+No offline operations (e.g. `load`/`dump`) can be performed yet.
+* `store copying` - The database is currently being updated from another instance of Neo4j.
```
Comment (jackwaudby) [inline modules/ROOT/pages/administration/databases.adoc]: Do we have docs on store copying we could link here? Feel free to ignore this comment.

File: modules/ROOT/pages/administration/databases.adoc [hunk 3]
```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
...
+* For other error states please refer to the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/manage-databases/errors/#database-management-states[operation manual].
```
Comment (jackwaudby) [inline modules/ROOT/pages/administration/databases.adoc]: I'd maybe drop the "error" here, I'm being pedantic but it sort of implies that the states above are also error states

File: modules/ROOT/pages/administration/databases.adoc (hunk 2 above)
Comment (gfx54b) [reply modules/ROOT/pages/administration/databases.adoc]: Did not find anything, what I can link

File: modules/ROOT/pages/administration/databases.adoc [hunk 4]
```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
 | BOOLEAN
 
 | requestedStatus
-| The expected status of the database. label:default-output[]
+| The expected status of the database. The value can be either `online` of `offline`. label:default-output[]
```
Comment (AlexicaWright) [inline modules/ROOT/pages/administration/databases.adoc]:
```suggestion
| The expected status of the database. 
The value can be either `online` or `offline`. label:default-output[]
```

File: modules/ROOT/pages/administration/databases.adoc [hunk 5]
```diff
@@ -180,11 +180,22 @@ The default for a standalone database is `neo4j://localhost:7687`. label:default
...
 | The actual status of the database. label:default-output[]
+
+The value can be one of the following:
+
+* `online` - The database is running.
+* `offline` - The database is not running.
+* `starting` - The database is not running, but is about to be.
+* `stopping` - The database is not running anymore, but still has not stopped completely.
+No offline operations (e.g. `load`/`dump`) can be performed yet.
+* `store copying` - The database is currently being updated from another instance of Neo4j.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/administration/databases.adoc]:
```suggestion
* `online` - the database is running
* `offline` - the database is not running
* `starting` - the database is not running, but is about to.
* `stopping` - the database is not running anymore, but still has not stopped completely.
Offline operations (e.g. `load`/`dump`) cannot be performed yet.
* `store copying` - the database is currently being updated from another instance of Neo4j
```

**Review by AlexicaWright — COMMENTED** [review]
Just a few editorial suggestions.

**Pull request merged**